### ISSUE TITLE
[Feature Request] Pagination Accessibility

### ISSUE THREAD

**Issue by JohnMica** [vuetifyjs/vuetify#9849]
### Problem to solve
Accessibility issues with the pagination buttons  / currently selected page

### Proposed solution
introduce options for the buttons to have aria-* attributes
introduce options for the page-numbers to have aria-* attributes

**Comment by jacekkarczmarczyk** [comment]
Please specify which exactly attributes should be added

**Comment by JohnMica** [comment]
seems that the least attributes needed are
- wrapping element `aria-label="Pagination Navigation"`
- buttons (prev / next) - `aria-label="Prev / Next page"`
- page numbers - `aria-label="page X"`
- current page - `aria-current="true"`

If there are supposed to be more attributes I will investigate and let you know about them. The ones mentioned above should be the minimum needed, I guess.

**Comment by JohnMica** [comment]
perhaps the values of aria-attributes could be editable - I'm thinking of language translations that could allow them to be modified ?

**Comment by jacekkarczmarczyk** [comment]
They should just go to language files https://github.com/vuetifyjs/vuetify/tree/master/packages/vuetify/src/locale

**Comment by abnersajr** [comment]
I would like to take this one. Is it possible to assign me to this one?
I had some previous experience with this.
I also agree that we can use `i18n` to the values.

**Comment by abnersajr** [comment]
PR is opened. Please feel free to review and request changes if necessary.

**Comment by wusongliang** [comment]
Thank you. Has the problem been solved?

**Comment by johnleider** [comment]
Closed via #9926

If you have any additional questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).
